ssl-config-generator/index.html

```diff
@@ -334,7 +334,11 @@
                     }
                     if (data.hstsEnabled == "true") {
                         data.hsts = '\n    # HSTS (ngx_http_headers_module is required) (15768000 seconds = 6 months)' + '\n' +
-                            '    add_header Strict-Transport-Security max-age=15768000;';
+                            '    add_header Strict-Transport-Security "max-age=15768000"';
+                        if (isSemVer(data.serverVersion, '>=1.7.5')) {
+                            data.hsts += ' always'
```

Comment:
It's not required, but might be nice to have a semicolon at the end of this statement.